suite-native/app/app.config.ts

```diff
@@ -61,6 +64,72 @@ if (isCI) {
     }
 }
 
+const getPlugins = (): ExpoPlugins => {
+    let plugins = [
+        [
+            'expo-font',
+            {
+                fonts: [
+                    '../../packages/theme/fonts/TTSatoshi-Medium.otf',
+                    '../../packages/theme/fonts/TTSatoshi-DemiBold.otf',
+                ],
+            },
+        ],
+        [
+            'expo-barcode-scanner',
+            {
+                cameraPermission: 'Allow $(PRODUCT_NAME) to access camera for QR code scanning.',
+            },
+        ],
+        [
+            'expo-build-properties',
+            {
+                android: {
+                    minSdkVersion: 28,
+                },
+                ios: {
+                    deploymentTarget: '14.0',
+                    flipper: true,
+                },
+            },
+        ],
+        '@trezor/react-native-usb/plugins/withUSBDevice.js',
+        // Define FLIPPER_VERSION
+        './plugins/withGradleProperties.js',
+        [
+            '@config-plugins/detox',
+            {
+                subdomains: '*',
+            },
+        ],
+    ];
+
+    // Debug build does not have access to the Sentry token.
+    if (buildType !== 'debug')
```

Comment:
Please use curly brackets for better readability.

Reply:
if you use the syntax suggested in this comment https://github.com/trezor/trezor-suite/pull/11821#discussion_r1548007789 then this comment is irrelevant...

Reply:
Fixed by 5d9543b375